index.js

```diff
@@ -189,15 +176,18 @@ class Hyperdiscovery extends EventEmitter {
   add (feed) {
     if (!feed.key) return feed.ready(() => { this.add(feed) })
     const key = datEncoding.toStr(feed.key)
-    const discoveryKey = datEncoding.toStr(feed.discoveryKey)
-    this._replicatingFeeds.set(discoveryKey, feed)
-
-    this.rejoin(feed.discoveryKey)
-    this.emit('join', { key, discoveryKey })
-    feed.isSwarming = true
+    const dKeyStr = datEncoding.toStr(feed.discoveryKey)
```

Comment:
```suggestion
    const dKeyStr = feed.discoveryKey.toString('hex')
```

Reply:
Is dat-encoding then used anywhere? If not we could remove it from package.json and require

Reply:
Might as well remove it entirely then. :D